common/src/main/java/dev/schmarrn/lighty/core/Compute.java

```diff
@@ -193,26 +200,27 @@ public static void computeCache(Minecraft minecraft) {
             }
 
             // as long as the section is in range...
-            if (!outOfRange(sectionPos)) {
-                // ... and the section is already compiled...
-                //if (!minecraft.levelRenderer.isSectionCompiled(sectionPos.origin())) {
-                //    // chunk data isn't ready yet, keep in queue
-                //    keepInUpdate.add(sectionPos);
-                //    continue;
-                //}
-                // ... we compute the new buffers, reduce the counter!
-                --ii;
-                cachedBuffers.compute(
-                        sectionPos,
-                        (pos, bufferHolder) -> buildChunk(
-                                renderer,
-                                dataProviders,
-                                pos,
-                                minecraft.level,
-                                bufferHolder != null ? bufferHolder : new BufferHolder()
-                        )
-                );
+            if (outOfRange(sectionPos)) {
+                continue;
             }
+            // ... and the section is already compiled...
+            //if (!minecraft.levelRenderer.isSectionCompiled(sectionPos.origin())) {
+            //    // chunk data isn't ready yet, keep in queue
+            //    keepInUpdate.add(sectionPos);
+            //    continue;
+            //}
+            // ... we compute the new buffers, reduce the counter!
```

Comment:
```suggestion
```

If I understand correctly, it is impossible now to have a section queued that isn't compiled yet.
If so, then please remove the commented out code